pyfar/classes/coordinates.py

```diff
@@ -1,9 +1,101 @@
-"""
-The following documents the pyfar coordinates class and functions for
-coordinate conversion. More background information is given in
-:py:mod:`coordinates concepts <pyfar._concepts.coordinates>`.
-Available sampling schemes are listed at
-:py:mod:`spharpy.samplings <spharpy.samplings>`.
+r"""
+The following introduces the the
+:py:func:`Coordinates class <pyfar.classes.coordinates.Coordinates>`
+and the coordinate systems that are available in pyfar. Available sampling
+schemes are listed at :py:mod:`spharpy.samplings <spharpy.samplings>`.
+
+Different coordinate systems are frequently used in acoustics research and
+handling sampling points and different systems can be cumbersome. The
+Coordinates class was designed with this in mind. It can convert between
+coordinate systems and rotate, query and plot coordinates points. Functions for
+converting coordinates not stored in a Coordinates object are available for
+convenience. However, it is strongly recommended to use the Coordinates class
+for all conversions.
+
+.. _coordinate_systems:
+
+Coordinate Systems
+------------------
+
+Each coordinate system has a unique name, e.g., `spherical_elevation`, and is
+defined by three coordinates, in this case `azimuth`, `elevation`, and
+`radius`. The available coordinate systems are shown in the image below.
+
+|coordinate_systems|
+
+.. _coordinates:
+
+Coordinates
+-----------
+
+The unit for length for the coordinates is always meter, while the unit for
+angles is radians. Each coordinate is unique, but can appear in multiple
+coordinate systems, e.g., the `azimuth` angle is contained in two coordinate
+systems (`spherical_colatitude` and `spherical elevation`). The table below
+lists all coordinates.
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Coordinate
```

Comment:
maybe we dont need to table anymore. The same content can be found below in the description of the properties.

Reply:
And we need to fix the math in the description of the properties.

Reply:
There's a little more information in the table, maybe extend the description of the properties?

Reply:
I like keeping both. This gives a very good overview.

Reply:
I guess fixing the math should be a separate pull, right?

Reply:
an other option could be to remove the table and order the properties, so then we would also have same information in a good order and we dont have the same information twice.
https://stackoverflow.com/questions/17193143/group-method-docstrings-in-sphinx

Reply:
I think not having this in alphabetical order might be confusing, but having information twice is also not ideal. We could also remove the information in the docstrings and link to the table.